src/background/devtools.js

```diff
@@ -27,6 +27,9 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
           console.info('[devtools] Clearing main local storage');
           chrome.storage.local.clear();
 
+          console.info('[devtools] Clearing sync storage');
+          chrome.storage.sync.clear();
```

Comment:
It is an debug option, but might be still unexpected. Maybe this can be guarded (clicking multiple times or something like that?).